node/coinstacks/common/api/src/evm/service.ts

```diff
@@ -357,16 +357,45 @@ export class Service implements Omit<BaseAPI, 'getInfo'>, API {
       gasUsed: tx.ethereumSpecific.gasUsed?.toString() ?? '0',
       gasPrice: tx.ethereumSpecific.gasPrice.toString(),
       inputData: inputData && inputData !== '0x' && inputData !== '0x0' ? inputData : undefined,
-      tokenTransfers: tx.tokenTransfers?.map((tt) => ({
-        contract: tt.contract,
-        decimals: tt.decimals,
-        name: tt.name,
-        symbol: tt.symbol,
-        type: tt.type,
-        from: tt.from,
-        to: tt.to,
-        value: tt.value,
-      })),
+      tokenTransfers: tx.tokenTransfers?.map((tt) => {
+        const value = (() => {
+          switch (tt.type) {
+            case 'ERC721':
+            case 'BEP721':
+              return '1'
+            case 'ERC1155':
+            case 'BEP1155':
+              return tt.multiTokenValues?.[0].value ?? '0'
```

Comment:
multi tokens can be multiple of the same id

Reply:
0 looks correct here, though do we want to guard against undefined `multiTokenValues[0]` as a paranoia check?
https://github.com/trezor/trezor-suite/blob/ddd24ef647b9d088c26c0d337cd2de3c9d4b1d56/suite-common/wallet-utils/src/transactionUtils.ts#L413-L416

Reply:
they are just checking length and then grabbing id, I think we are good unless we see the api change

Reply:
meh, doesn't hurt, adding lol